crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -442,7 +442,7 @@ impl CodeGenerateable for EsmExports {
                 EsmExport::LocalBinding(name, mutable) => {
                     if *mutable {
                         Some(quote!(
-                            "([() => $local, (v) => $local = v])" as Expr,
```

Comment:
The previous code fails if the name of the top-level variable is `v`.

This was not a problem for almost all modules, but some packages like [`@firebase/webchannel-wrapper`](https://unpkg.com/@firebase/webchannel-wrapper@0.2.41/dist/index.esm.js) is published after mangling, so it can be a problem

Reply:
Now it fails for `___v`. Could we name it `name + "_new_value"`?